test/testnet_scion/README.md

```diff
@@ -109,20 +109,22 @@ Install the scion module:
 go build -o ./bin/ ./go/scion/
 ```
 
+Ping with LF using IPv4:
```

Comment:
This sounds as if this is an IPv4 ping. However, it is a SCION ping to an IPv4 host address. Not really sure what a better way would be to describe this. Nevertheless here a suggestion.
```suggestion
SCION ping (IPv4) with LF:
```

Reply:
Makes sense. I went with your suggestion.